ecommerce/templates/oscar/basket/basket.html

```diff
@@ -9,7 +9,7 @@
 {% endblock title %}
 
 {% block navbar %}
-    {% include 'edx/partials/_student_navbar.html' with hide_user_id=True %}
+    {% include 'edx/partials/_student_navbar.html' %}
```

Comment:
Similar comment here as above; I saw that you are overriding the entire file basically rather than just the block - is that necessary?